tests/integrations/test_sklearn.py

```diff
@@ -90,91 +83,6 @@ def dummy_data(request):
     return (data, labels)
 
 
-def test_learning_curve(dummy_classifier):
```

Comment:
Are these tests no good?   Duplicating functional and unittests is a perfectly valid thing to do.

But if the unittests are poor, im fine with removing them, i havent looked carefully enough to make a judgement.  what was the reasoning for removing?

Reply:
Most were testing at a lower level than we'd like to test -- testing functionality in the midlevel of the API, namely `roc`, `learning_curve`, `confusion_matrix` and `precision_recall`. I suspect they tested this level because it was infeasible, without `yea`, to test the top level of the API, which calls `wandb.log`.

Reply:
Thank you for explaining!  looks good